spec/index.bs

```diff
@@ -15952,19 +15954,22 @@ The [=texel block memory cost=] of each of these formats is the same as its
             <th>[=Texel block copy footprint=] (Bytes)
             <th>{{GPUTextureSampleType}}
             <th>Texel block [=texel block width|width=]/[=texel block height|height=]
+            <th>Sliced 3D
```

Comment:
It's a little odd to say rgb9e5ufloat supports "Sliced 3D" since it's a 1x1 format. How about:

```suggestion
            <th>{{GPUTextureDimension/"3d"}}
```

Reply:
Makes sense, thank you!